### PR TITLE
OCPQE-22600: add new cluster profile azure-autorelease-qe

### PR DESCRIPTION
**Pull request by jinyunma**
https://issues.redhat.com/browse/OCPQE-22600

### CODE REVIEW

**Comment by jinyunma** [comment]
cc @jianlinliu to review

**Comment by jinyunma** [comment]
@droslean could you help to review? thanks.

**Comment by droslean** [comment]
/hold

@psalajova Is the cluster profile configuration in production already?

File: pkg/api/types.go [hunk 1]
```diff
@@ -1794,6 +1797,8 @@ func (p ClusterProfile) LeaseType() string {
 		return "azuremag-quota-slice"
 	case ClusterProfileAzureQE:
 		return "azure-qe-quota-slice"
+	case ClusterProfileAzureAutoreleaseQE:
+		return "azure-autorelease-qe"
```
Comment (psalajova) [inline pkg/api/types.go:1801]:
```suggestion
		return "azure-autorelease-qe-quota-slice"
```

**Review by psalajova — COMMENTED** [review]
https://docs.ci.openshift.org/docs/how-tos/adding-a-cluster-profile/#adding-new-leases

**Comment by jianlinliu** [comment]
@droslean can we unload this PR to move forwards?

**Comment by droslean** [comment]
/lgtm

@jinyunma I don't see any secret created for this cluster profile. Even if we merge this PR the profile is not useable. Please try to follow https://docs.ci.openshift.org/docs/how-tos/adding-a-cluster-profile/

**Comment by jinyunma** [comment]
> @jinyunma I don't see any secret created for this cluster profile. Even if we merge this PR the profile is not useable. Please try to follow https://docs.ci.openshift.org/docs/how-tos/adding-a-cluster-profile/

secret file is created by PR https://github.com/openshift/release/pull/52762, and waiting for this PR merged firstly.

**Pull request merged**